server/src/test/scala/za/co/absa/atum/server/api/service/DatabaseServiceTest.scala

```diff
@@ -0,0 +1,28 @@
+package za.co.absa.atum.server.api.service
+
+import org.scalatest.funsuite.AnyFunSuite
+import org.mockito.MockitoSugar
+import za.co.absa.atum.model.dto.CheckpointDTO
+import za.co.absa.atum.server.api.provider.PostgresAccessProvider
+import scala.concurrent.Future
+
+class DatabaseServiceTest extends AnyFunSuite with MockitoSugar {
+
+  test("testSaveCheckpoint should save a checkpoint in the database")  {
+    val mockPostgresAccessProvider = mock[PostgresAccessProvider]
+    val checkpointDTO = mock[CheckpointDTO]
+    val databaseService = new DatabaseService()
+
+    when(mockPostgresAccessProvider.runs.writeCheckpoint(checkpointDTO)).thenReturn(Future.successful(checkpointDTO))
+
+    val results = databaseService.saveCheckpoint(checkpointDTO)
+    println(results)
+    results.complete(checkpointDTO)
+
+  }
+
+  test("testReadCheckpoint") {}
+
+  test("testPostgresAccessProvider") {}
```

Comment:
this doesn't really test anything, is it still in progress?

Reply:
Yes, it was just a preparation and I forgot to remove them because the implementation was totally wrong.